docs/core/compatibility/2.2-3.1.md

```diff
@@ -61,11 +61,11 @@ If you're migrating from version 2.2 to version 3.1 of .NET Core, ASP.NET Core,
 - [SPAs: SpaServices and NodeServices console logger fallback default change](#spas-spaservices-and-nodeservices-no-longer-fall-back-to-console-logger)
 - [Target framework: .NET Framework not supported](#target-framework-net-framework-support-dropped)
 
-[!INCLUDE[HTTP: Browser SameSite changes impact authentication](~/includes/core-changes/aspnetcore/3.1/http-cookie-samesite-authn-impacts.md)]
+[!INCLUDE[HTTP: Browser SameSite changes impact authentication](/includes/core-changes/aspnetcore/3.0/authn-apis-json-types.md)]
```

Comment:
I think @scottaddie is out this week, but it looks like you're change the include file here. Can you explain?